src/Root.tsx

```diff
@@ -0,0 +1,23 @@
+import {
+  HashRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { App } from './App';
+
+import { Home } from './components/Home';
+import { People } from './components/People';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<Home />} />
+        <Route path="people/:selected?" element={<People />} />
+        <Route path="*" element={<h1 className="title">Page not found</h1>} />
```

Comment:
Create component for `NotFountPage`

Reply:
No problem. But is it necessary? I mean it just a half row long and consist of one <h1> tag. Does it really needs separate component?

Reply:
But your `HomePage` can grow after some time, and you will need to move it to separate component 😉